openpype/hosts/maya/startup/userSetup.py

```diff
@@ -13,23 +12,40 @@
 
 print("Starting OpenPype usersetup...")
 
+settings = get_project_settings(os.environ['AVALON_PROJECT'])
+
+# Loading plugins explicitly.
+if settings["maya"]["explicit_plugins_loading"]["enabled"]:
+    def _explicit_load_plugins():
+        project_settings = get_project_settings(os.environ["AVALON_PROJECT"])
```

Comment:
This shouldn't get the project settings again but use the project settings that were already acquired outside of this function.

Reply:
Yup, done